### ISSUE TITLE
'Token edit tool' for annotation

### ISSUE THREAD

**Issue by B-CARON** [Arborator/arborator-frontend#223]
- When moving the index to the sentence in order to select a token to edit (generally, in my case, to delete), a flag appears 
 than can hide the tokens , preventing their selection. 

- This same tool does not work for Dominque Caubet (@casanaija), who cannot edit any token in her DarijaDom project. 

- When adding or editing a token, all the features of the sentence are deleted. 

-

**Comment by kirianguiller** [comment]
yes, this feature is not working correctly, and should be deleted in my opinion (no one want to be able to add or edit tokens at the price of deleting all information about the tree (dependency, rel, upos, feats, etc).

As we can already edit a single token by clicking on it, what is missing now is :
- to be able to insert a new token at a given index
- to be able to delete current tokens (the easiest way is to have a "delete this token" button inside the (single) token edit dialog (the dialog that currently appears when clicking on a token)

if these two feature are separate, it will be easier to code the fonctionality and be able to preserve the information about the full tree. Indeed, the problem with the current "edit tokens" feature is that we don't know what token index was inserted/edited/deleted, which means that we can't  keep the current tree and just add/edit/remove corresponding tokens

**Comment by khansadaoudi** [comment]
We changed the UI of this feature, if you want to (delete, split, insert, merge with the right or left token), you have to select only the token that you want modifiy and you will get this list of options.
